### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added Readme file

### CODE REVIEW

**Review by itsmebipul00 — APPROVED** [review]
Nice work, good to merge

**Review by Naman-Saxena1 — APPROVED** [review]
Can add 2-3 screenshots of components in Library. Otherwise this looks great 👍

File: README.md [hunk 1]
```diff
@@ -1,2 +1,250 @@
...
+Check it out at: https://xendesign.netlify.app/components/grid.html
+
+---
+
+### Slider
+
+XenDesign provides the following types of Slider:
+
+- Custom Standard Slider
+
```
Comment (eldhoabraham94) [inline README.md:263]: The Readme seems quite comprehensive. Good to merge

**Review by eldhoabraham94 — APPROVED** [review]
The Readme seems quite comprehensive. Good to merge

**Review by bossmonkey98 — APPROVED** [review]

**Pull request merged**